internal/auth0/tenant/flatten.go

```diff
@@ -16,8 +16,6 @@ func flattenTenant(data *schema.ResourceData, tenant *management.Tenant) error {
 		data.Set("support_email", tenant.GetSupportEmail()),
 		data.Set("support_url", tenant.GetSupportURL()),
 		data.Set("allowed_logout_urls", tenant.GetAllowedLogoutURLs()),
-		data.Set("session_lifetime", tenant.GetSessionLifetime()),
```

Comment:
We could keep these and then completely remove the elses below.

Reply:
We could, but we'd potentially be setting the value twice. If an error occurs while setting the first, but the second succeeds, there will still be an error.

Reply:
Extremely unlikely that we'd incurr an error considering we're using the safe getters.

Reply:
Also if we don't want to do it twice we can make use of a variable instead. 

```go
sessionLifeTime := tenant.GetSessionLifetime()
if sessionLifeTime == 0 {
  sessionLifeTime = default
}

data.Set(...)
```